app/models/user.rb

```diff
@@ -89,6 +87,7 @@ class User < ApplicationRecord
     dependent: :destroy
   has_many :following, class_name: 'Follow', foreign_key: 'follower_id',
     dependent: :destroy
+  has_many :linked_profiles
```

Comment:
Pretty much all `has_many` associations ought to have `dependent: :destroy`.  I forgot about this until recently, but it's important, because otherwise things will break if we try to delete a user